jdbc/src/main/java/org/apache/metamodel/jdbc/JdbcDataContext.java

```diff
@@ -289,6 +289,13 @@ private boolean usesCatalogsAsSchemas(DatabaseMetaData metaData) {
         } catch (SQLException e) {
             throw JdbcUtils.wrapException(e, "retrieve schema and catalog metadata", JdbcActionType.METADATA);
         } finally {
+            if(rs != null) {
```

Comment:
Very minor, but our code style conventions are to have a space after `if` and before the `(`

Reply:
Actually, even better would be for you to just use `FileHelper.safeClose(rs);` instead of this if + try-catch stuff.